Gemfile.lock

```diff
@@ -52,6 +52,3 @@ DEPENDENCIES
   cucumber (~> 1.3.9)
   ronn
   sinatra
-
-BUNDLED WITH
```

Comment:
Thanks for the tip @mislav! I guess we are always learning new and better ways to do things.